charts/gitops-server/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  # FIXME check the app name
+  repository: ghcr.io/weaveworks/wego-app
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.6.2"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+logLevel: info
+# Additional arguments to pass in
+# additionalArgs: []
+
+# To enable enterprise OIDC:
+# envVars:
+# - name: WEAVE_GITOPS_AUTH_ENABLED
+#   value: true
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  # Specifies whether the clusterRole & binding to the service account should be created
+  create: true
+  # List of resources that the service account can impersonate
+  impersonationResourceNames: []
+  # Limit the type of principal that can be impersonated
+  # impersonationResources: [ "users", "groups" ]
+  # Specify whether additional permissions should be granted to the service account
+  # in order to work with weave-gitops enterprise
+  viewSecrets: []
+
+# This should not be used in production
```

Comment:
this will be used in production

Reply:
I'm unclear as to why this would be used in production, is there a ticket for clarifying/resolving it's use there?

Reply:
see that thread I linked. the AdminUser feature is a deliberate one for prod.
I have to write up the tickets now

Reply:
cf. above thread, is there a reason not to change this when those changes are implemented?

Reply:
you mean when https://github.com/weaveworks/weave-gitops/issues/1787 is done? yep makes sense